pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -339,6 +356,20 @@ func (r *ReconcileNetworkAddonsConfig) applyObjects(networkAddonsConfig *opv1alp
 	return nil
 }
 
+// Delete removed objects
+func (r *ReconcileNetworkAddonsConfig) deleteObjects(networkAddonsConfig *opv1alpha1.NetworkAddonsConfig, objs []*unstructured.Unstructured) error {
+	for _, obj := range objs {
+		// Apply all objects on apiserver
```

Comment:
Outdated doc, you don't apply, you delete. I think it is redundant in this tiny function, let's drop it.